constants.go

```diff
@@ -613,6 +613,14 @@ const (
 	// reading cluster events and playing back session records.
 	PresetAuditorRoleName = "auditor"
 
+	// PresetReviewerRoleName is a name of a preset role that allows
+	// for reviewing access requests.
+	PresetReviewerRoleName = "reviewer"
+
+	// PresetRequesterRoleName is a name of a preset role that allows
+	// for requesting resources.
```

Comment:
```suggestion
	// for requesting access to resources.
```

Though if these roles are only for Okta stuff, we should give them more descriptive names and godocs.

Reply:
The use case is primarily Okta, but they're more general than that.